mavlink/test_mission_ended.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import time
+
+import dronekit
+
+from modules import flight_controller
+
+DELAY_TIME = 10.0  # seconds
+MISSION_PLANNER_ADDRESS = "tcp:127.0.0.1:14550"
+TIMEOUT = 1.0  # seconds
+
+MAVLINK_TAKEOFF_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_TAKEOFF_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_TAKEOFF
+MAVLINK_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT
+
+ALTITUDE = 10
+ACCEPT_RADIUS = 10
+
+
+def upload_mission(
```

Comment:
A large amount of code in the pathing repository should be moved to the common repository. In particular, any code that interacts directly with Dronekit-Python.